# Instrumentation Design 1
## Lab 3 - Parallel Series Circuit Combinations: Kircoff's voltage and current laws

Lab report by Matt Rixman

### 1. Resistance of Series-Parallel circuits

Objective 1: To verify experimentally the rules for finding the total resistance Rt of a series-parallel circuit.

Objective 2: To design a series-parallel network that will meet specified current requirements.

The following resistors were used in the procedures below:

In [32]:
from si_prefix import si_format as f # used to generate the tables below
k = 1000
r1 = 330  # Ω
r2 = 470
r3 = 560
r4 = 1.2 * k
r5 = 2.2 * k
r6 = 3.3 * k
r7 = 4.7 * k
r8 = 10 * k

# for calculating the resistance of resistors in parallel
def parallel_2(a, b):
    return a * b / (a + b)

def parallel_3(a, b, c):
    return a * b * c / (a * b + b * c + a * c)

1. The above 8 resistors were connected to match figure 1.7
2. Measured Resistances:
| From Point | To Point |  Measured Resistance (Ω) |
|:-|:-|:-|
| **B** | **C** | 332.7 |
| **A** | **D** | 1.23k |
| **A** | **G** | 12.82k |


3. Calculated total resistance (A-G) (Ω):
$$
R_{AG} = R_1 + \frac{R_2 R_4}{R_2 + R_4} + R_3 + \frac{R_5 (R_6 + R_7)}{R_5 + R_6 + R_7} + R_8
$$

In [33]:
r_tot = r1 + parallel_2(r2, r4) + r3 + parallel_2(r5, r6 + r7) + r8
f(r_tot, precision=2) + 'Ω'

'12.95 kΩ'

4. The total measured resistance was 12.82kΩ which compares favorably with the calculated value of 12.95kΩ
5. Starting from the layout in fig 1.9 and targeting 5 mA at 10V, we chose to replace $R_2$ with $R_9$

In [34]:
r9 = 200 # Ω

$$
R_{calculated} = R_1 + \frac{R_5 (R_6 + R_4)}{R_5 + (R_6 + R_4)} + R_9
$$

In [35]:
r_tot = r1 + parallel_2(r5, r6 + r4) + r9
f(r_tot, precision=0) + 'Ω'

'2 kΩ'

6. The circuit described in step 5 measured 1.96kΩ, and drew 4.90mA when 10v was applied

### 2. Kirchoff's Voltage Law

Objective 1: To find a relationship between the sum of the voltage drops across series-connected resistors, and the applied voltage.

Objective 2: To verify experimentally the relationship found in objective 1


1. Using the resistor values from the previous section, we constructed a circuit similar to figure 2.4.  We anticipate its resistance will be:


$$
R_{calculated} = R_1 + R_2 + R_3 + R_4
$$

In [36]:
r_tot = r1 + r2 + r3 + r4
f(r1 + r2 + r3 + r4, precision=2) + 'Ω'

'2.56 kΩ'

2. At 15V the total current should be 
$$
I = \frac{15V}{2.56\text{k}\Omega} = 5.86\text{mA}
$$

This current passes through every resistor (since they're in series), so the voltage drop at any given resistor should be:

$$
V_i = 5.86\text{mA} \times R_i
$$

In [37]:
m = .001
def drop_3(ri):
    return 5.86 * m * ri


3. The above calculations compare favorably with the measured ones

| Resistor | Resistance(Ω) | Calculated Voltage Drop (V) | Measured Voltage Drop (V) |
|:-|:-|:-|:-|
| r1 | {{ f(r1, precision=0) }} |{{ f(drop_3(r1), precision=2) }} | 1.94 |
| r2 | {{ f(r2, precision=0) }} |{{ f(drop_3(r2), precision=2) }} | 2.75 |
| r3 | {{ f(r3, precision=0) }} |{{ f(drop_3(r3), precision=2) }} | 3.29 |
| r4 | {{ f(r4, precision=0) }} |{{ f(drop_3(r4), precision=2) }} | 7.07 |

4. Repeating this procedure for the circuit in figure 2.5 gives a total resistance of

$$
R_{calculated} = R_1 + \frac{R_2 R_3}{R_2 + R_34} + \frac{R_5 R_6 R_7}{R_5 R_6 + R_6 R_7 + R_5 R_8}
$$

In [38]:
r_tot = r1 + parallel_2(r2, r3) + r4 + parallel_3(r5, r6, r7) + r8
f(r_tot, precision=2) + 'Ω'

'12.82 kΩ'

At 15V the total current should be 
$$
I = \frac{15V}{12.82\text{k}\Omega} = 1.17\text{mA}
$$

This current passes through every resistor (since they're in series), so the voltage drop at any given resistor should be:

$$
V_i = 1.17\text{mA} \times R_i
$$

In [39]:
m = .001
def drop_4(ri):
    return 1.17 * m * ri

|Measuring At| Across Resistors | Resistance(Ω) | Calculated Voltage Drop (V) | Measured Voltage Drop (V) |
|:-|:-|:-|:-|:-|
| V1 | r1 | {{ f(r1, precision=0) }} |{{ f(drop_4(r1), precision=0) }} | 385m |
| V2 | r2, r3 | {{ f(parallel_2(r2, r3), precision=0) }} |{{ f(drop_4(parallel_2(r2, r3)), precision=0) }} | 298m |
| V3 | r4 | {{ f(r4, precision=1) }} |{{ f(drop_4(r4), precision=2) }} | 1.40 |
| V4 | r5, r6, r7 | {{ f(parallel_3(r5, r6, r7), precision=1) }} |{{ f(drop_4(parallel_3(r5, r6, r7)), precision=2) }} | 1.20 |
| V5 | r8 | {{ f(r8, precision=0) }} |{{ f(drop_4(r8), precision=2) }} | 11.77 |

### 3. Kirchoff's Current Law

Objective 1: To find a relationship between the sum of the currents entering any junction of an electric circuit and the current leaving that junction.

Objective 2: To verify experimentally the relationship found in objective 1

1. We applied 15 Volts to the circuit from Section 2, step 4 (Figure 3.4).


2. The following currents were measured:
|Measuring At| After Resistor(s) | Resistance(Ω) | Measured Current (A) |
|:-|:-|:-|:-|
| I<sub>TA</sub> | r1 | {{ f(r1, precision=0) }} | 1.18m |
| I<sub>TB</sub> | r2, r3 | {{ f(parallel_2(r2, r3), precision=0) }} | 298m |
| I<sub>TC</sub> | 1.18m | {{ f(r4, precision=1) }} | 1.40 |
| I<sub>TD</sub> | r5, r6, r7 | {{ f(parallel_3(r5, r6, r7), precision=0) }} | 1.18m |
| I<sub>TE</sub> | r8 | {{ f(r8, precision=0) }} | 1.18 m|
| I<sub>2</sub> | r2 | {{ f(r2, precision=0) }} | 586µ |
| I<sub>3</sub> | r3 | {{ f(r3, precision=0) }} | 490µ |
| I<sub>5</sub> | r5 | {{ f(r5, precision=1) }} | 547µ |
| I<sub>6</sub> | r6 | {{ f(r6, precision=1) }} | 360µ |
| I<sub>7</sub> | r7 | {{ f(r7, precision=1) }} | 253µ |

I<sub>2</sub> + I<sub>3</sub> = {{ f(10**-6 * (586 + 490), precision=2) }}A

I<sub>5</sub> + I<sub>6</sub> + I<sub>7</sub> = {{ f(10**-6 * (547 + 360 + 253), precision=2) }}A

3. Targeting 6mA of current draw at 15V, we endeavored to design a circuit with a total reistance of
$$
\frac{15\text{V}}{6\text{mA}} = 2.5\text{k}\Omega
$$

The following resistors achieve the target resistance

$$
330\Omega + \frac{1}{
    \frac{1}{1.7\text{k}\Omega} + \frac{1}{5.1\text{k}\Omega} + \frac{1}{3.4\text{k}\Omega}
    } + 1.2\text{k}\Omega = 2.5\text{k}\Omega 
$$

4. The current through each resistor was measured, and appears in the diagram:

![](circuit1.png)

The three parallel resistors were chosen such that R<sub>2</sub> is half of R<sub>4</sub>, and R<sub>3</sub> is thrice R<sub>2</sub>.  The other two resistors were chosen to achieve to desired total resistance.